iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.m

```diff
@@ -1204,9 +1204,11 @@ + (NSDictionary *)sizeToDictionary:(CGSize)size
 + (NSDictionary *)touchPropertiesToDictionary:(UITouch *)touch andView:(UIView *)view
 {
   if ([self forceTouchSupported] || [self validatePencilWithTouch:touch]) {
```

Comment:
In case if condition failed, it will return the old value. That should also be updated .

Reply:
Oh I see. You're talking about the `pointToDictionary()` call down below. I'll update it. Thanks.

Reply:
PR updated.